python/ray/rllib/agents/sac/sac/custom_model.py

```diff
@@ -0,0 +1,184 @@
+from collections import OrderedDict
+from typing import Tuple, Dict, Iterable, Union, Optional, Sequence, Callable
+
+import gym
+import numpy as np
+import tensorflow as tf
+
+from sac import config
+from sac.rllib_proxy import TFModelV2, normc_initializer, get_activation_fn
+
+ArrayShape = Tuple[int, ...]
+KernelInitializer = Callable[[], tf.keras.initializers.Initializer]
+
+
+def build_fcn(
```

Comment:
Question - So later on when we support Luminance, we need to have a similar one `def build_convn()` right?

Reply:
For SAC, yes.